portfolio/src/main/webapp/index.html

```diff
@@ -2,145 +2,142 @@
 <html>
 
 <head>
-  <meta charset="UTF-8">
-  <title>Grant Commodore's Portfolio Project</title>
-  <link rel="stylesheet" href="style.css">
-  <script src="script.js"></script>
+	<meta charset="UTF-8">
```

Comment:
This kind of change (formatting) generally shouldn't be "bundled" into another change because I now can't see what has changed easily. Take a look at the [diff on github and see for yourself](https://github.com/GrantComm/STEP/pull/7/files#diff-a139a4973391a46a75b034736e4d6876). I think you've done it here though because you were already in the middle of something else when you did this. In the future, I'd suggest checking out another branch, making this change, then rebasing your current branch on top of the formatting change.

```
  # create a new branch (from master) and make formatting changes
  git checkout master && git branch formatting
  # rebase your feature branch onto the formatting branch so you have the changes locally (but they're separately reviewable)
  git rebase formatting whatever-this-branch-is-called
```

Commit and send out formatting for review, then when it's submitted,

```
  # rebase this branch onto master now that formatting changes are in 
  git pull origin master
  git rebase master whatever-this-branch-is-called
  git push origin whatever-this-branch-is-called --force
```

Since the changes are already in this commit, what I would probably do would be to duplicate them into another branch, then do the rebase and whatnot. I think git can handle duplicate changes like this without getting upset and requiring a difficult branch conflict resolution.